.github/workflows/pull_request.yml

```diff
@@ -155,7 +157,7 @@ jobs:
 
       - name: Build
         working-directory: arrow-libs
-        run: ./gradlew macosX64Test iosX64Test
+        run: ./gradlew macosX64Test iosX64Test macosArm64Test iosArm64TestKlibrary iosArm64TestKlibrary iosArm32TestBinaries iosArm32TestKlibrary iosSimulatorArm64Test
```

Comment:
Do these tasks not have a `xxxTest` variant i.g is `iosArm64Test` not a task? And why do we need both `iosArm64TestKlibrary` and `iosArm64TestKlibrary`?

Reply:
Thanks for that catch. 

```
Do these tasks not have a xxxTest variant i.g is iosArm64Test not a task?
```
Yes, there aren't yet gradle test task for those, so we do this instead to run binary tests and library tests